tools/oesign/main.c

```diff
@@ -578,6 +578,8 @@ int arg_handler(int argc, const char* argv[])
     return ret;
 }
 
+/* make main() weak (programs that link oesignlib may override it) */
+OE_WEAK
```

Comment:
Instead of making this WEAK, the main function could just be excluded from the library.

Reply:
done